install.sh

```diff
@@ -24,8 +24,8 @@ then
 fi
 
 # Check for Xcode minimum version
-XCODE_MIN_VERSION=130
-XCODE_MIN_VERSION_STR="Xcode 13.0"
+XCODE_MIN_VERSION=160
```

Comment:
I think this and the readme got out of sync with the min version spreadsheet, assuming we want Xcode 16 for 13.0?